runner/prepare_to_execute_test.go

```diff
@@ -1,84 +1,87 @@
 package runner
 
 import (
-	"fmt"
 	"runtime"
 	"testing"
+
+	"github.com/metrumresearchgroup/wrapt"
 )
 
-func TestPrepareForExecution(t *testing.T) {
-	var pathtests []struct {
-		in  []string
-		out []string
+func TestPrepareForExecution(tt *testing.T) {
+	type pathtest struct {
+		name string
+		in   []string
+		out  []string
 	}
+
+	var tests []pathtest
+
 	if runtime.GOOS == "windows" {
```

Comment:
May make sense to make this build specific to windows or move to a `_windows_test.go` file later to keep things clean

Reply:
I was thinking the same, but at the same time I don't have a windows machine to run it on.